recipe/meta.yaml

```diff
@@ -1,19 +1,19 @@
 package:
     name: gdal
-    version: 1.11.4
+    version: 2.0.2
 
 source:
-    fn: gdal-1.11.4.tar.gz
-    url: http://download.osgeo.org/gdal/1.11.4/gdal-1.11.4.tar.gz
-    md5: e9a966f0f5a88c43da47faf78a865952
+    fn: gdal-2.0.2.tar.gz
+    url: http://download.osgeo.org/gdal/2.0.2/gdal-2.0.2.tar.gz
+    md5: 573865f3f59ba7b4f8f4cddf223b52a5
     patches:
         - clang.patch  # [osx]
         - prepend-dll.patch  # [win]
-        - MSC_VER_1900_snprintf.patch  # [win]
         - install_scripts.patch
 
 build:
     number: 0
+    skip: True # [not osx]
```

Comment:
@kmuehlbauer Why are we skipping all but `osx`? If I recall correctly Windows will fail only on PY35 (that is OK), but Linux was build correctly, right?